### DISCUSSION TITLE
DM-1701 Callsign DB

### DISCUSSION THREAD

**Discussion by kliamr** [hmatuschek/qdmr#376] [Q&A] [answered]
Hi all,

Thank you hmatuschek for enabling me to get around my old PC, old Windows and old Excel limitations for uploading 120K entries to the DM-1701 ContactsCSV. The ability to specify prefixes is a great help in trimming it down.

It works very well but the DM-1701 only displays the callsign, name and country fields. Is there any way of also displaying the city and state fields? The way it does when imported from Excel, although I am limited to 65,536 entries with the Excel import.

Not a big / important issue but would be nice to have if there is any way of doing this that I am missing.

Thanks all.

**Comment by hmatuschek** [comment, marked as answer]
Created an issue for that (#377), so I don't forget about it. Currently, I use the same call-sign DB format for many TYT (and similar) radios. Currently, only name, surname and country are encoded, as the string field for each entry is limited to 100bytes of ASCII. Of cause, I could extend it.

**Reply by kliamr** [reply]
Thank you, much appreciated.

**Reply by hmatuschek** [reply]
Pushed some changes to the https://github.com/hmatuschek/qdmr/tree/377-fix-tyt-callsign-db branch, that should extend the call-sign entries to cities, states and comments.

**Reply by kliamr** [reply]
Thank you Hannes, it works perfectly and now displays the additional details.

**Reply by allesand** [reply]
@hmatuschek Would the "auto limit number of entries" feature work here the same way it does with the Anytone radios?

@kliamr On my Anytone with space for 200k entries I do not need to limit them manually, since qdmr does that based on my DMR id. So if you just need the maximum number of possible entries "close to you", you can just write the user DB and let qdmr do the limiting on the max number of entries the radio can hold, if that works the same way on the DM-1701

https://dm3mat.darc.de/qdmr/manual/ch02s11.html -> "Call-sign DB settings"

If you want "far away" IDs included, manually selecting prefixes works better, though.

**Reply by kliamr** [reply]
Yes, manually selecting prefixes works for me as I want the likes of Australia and New Zealand to be included.
